tests/contrib/hooks/test_jdbc_hook.py

```diff
@@ -47,7 +47,7 @@ def test_jdbc_conn_connection(self, jdbc_mock):
         jdbc_conn = jdbc_hook.get_conn()
         self.assertTrue(jdbc_mock.called)
         self.assertIsInstance(jdbc_conn, Mock)
-        self.assertEqual(jdbc_conn.name, jdbc_mock.return_value.name)
+        self.assertEqual(jdbc_conn.name, jdbc_mock.return_value.name)  # pylint: disable=no-member
```

Comment:
Similar to the migration -- are we not filtering out our pylint-todo.txt anymore?

Reply:
Will check.

Reply:
I reverted migration changes but test_jdbc_hook was beacause now we are using newer version of pylint (previously we fixed it a 2.3.1. Now pylint from setup.py is used and not fixed. test_jdbc_hook.py was not on the pylint_todo.txt list :(